### ISSUE TITLE
Resistor icon is not appearing on site

### ISSUE THREAD

**Issue by kenliu** [exercism/exercism#4791]
Ruby track. See screenshot. 
![image](https://user-images.githubusercontent.com/47234/55525434-065ab300-565f-11e9-978d-442c641d7a9d.png)

**Comment by kotp** [comment]
![resistor_colors_on_exercise](https://user-images.githubusercontent.com/16747/55526197-14f69980-5662-11e9-8fc1-9479abf9f807.png)

and 

![resistor_colors_on_tracks_ruby](https://user-images.githubusercontent.com/16747/55526236-32c3fe80-5662-11e9-85f7-1352455a5fba.png)

**Comment by rpottsoh** [comment]
Exercise Resistor-Color has an icon Resistor-Colors does not.  I believe the latter exercise name will be changing to help differentiate it from Resistor-Color.  See exercism/problem-specifications#1495

**Comment by rpottsoh** [comment]
The renamed Resistor-Colors name will require an icon.

**Comment by iHiD** [comment]
Yes - I'm waiting on this rename before adding the images. `resistor-color` should work correctly.

**Comment by kotp** [comment]
I had to post the additional pictures, as it was confusing enough with no named label in the "You've unlocked" portion.

**Comment by iHiD** [comment]
Is this fixed?
